update_requirements.sh

```diff
@@ -39,4 +39,5 @@ pyenv rehash
 
 echo "precommit install ..."
 pre-commit install
-[ -d "./xx" ] && cd xx && pre-commit install
\ No newline at end of file
+[ -d "./xx" ] && cd xx && pre-commit install
+[ -d "./notebooks" ] && cd xx && pre-commit install
```

Comment:
does this work? current directory should already be xx, and think you want to be in notebooks, right?

Reply:
yeah you're right